### PR TITLE
[relay] Feature/relay remove cfg file

### PR DESCRIPTION
**Pull request by pappz**
## Describe your changes

- Remove config file support
- Add ENV variable support

### Checklist
- [ ] Is it a bug fix
- [ ] Is a typo/documentation fix
- [x] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: relay/cmd/main.go [hunk 1]
```diff
@@ -76,7 +73,6 @@ var (
 func init() {
 	_ = util.InitLog("trace", "console")
```
Comment (lixmal) [inline relay/cmd/main.go:75]: Do we still need this?

File: relay/cmd/main.go [hunk 2]
```diff
@@ -85,7 +81,11 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&cobraConfig.LetsencryptAWSRoute53, "letsencrypt-aws-route53", false, "use AWS Route 53 for Let's Encrypt DNS challenge")
 	rootCmd.PersistentFlags().StringVarP(&cobraConfig.TlsCertFile, "tls-cert-file", "c", "", "")
 	rootCmd.PersistentFlags().StringVarP(&cobraConfig.TlsKeyFile, "tls-key-file", "k", "", "")
-	rootCmd.PersistentFlags().StringVarP(&cobraConfig.AuthSecret, "auth-secret", "s", "", "log level")
+	rootCmd.PersistentFlags().StringVarP(&cobraConfig.AuthSecret, "auth-secret", "s", "", "auth secret")
+	rootCmd.PersistentFlags().StringVar(&cobraConfig.LogLevel, "log-level", "info", "log level")
+	rootCmd.PersistentFlags().StringVar(&cobraConfig.LogFile, "log-path", "console", "log path")
```
Comment (lixmal) [inline relay/cmd/main.go]: should this be `log-file` to be consistent with mgmt server etc?

File: relay/cmd/main.go [hunk 3]
```diff
@@ -85,7 +81,11 @@ func init() {
...
+
+	setFlagsFromEnvVars(rootCmd)
```
Comment (lixmal) [inline relay/cmd/main.go:89]: The order usually is env -> flags, i.e. cli flags override any env vars. At least that's how aws & friends handle it. Now I must admit I'm unsure how we handle it in our other components, maybe that should take precedence.

Comment (pappz) [reply relay/cmd/main.go:89]: You are right but this code does what you expect.
In this example the listen address will be :1234
```
NB_LISTEN_ADDRESS=:654 ./cmd --listen-address=:1234 
```
But fix me if I mixed something.

File: relay/cmd/main.go (hunk 1 above)
Comment (pappz) [reply relay/cmd/main.go:75]: I think so. If we have any issues before line 104. (where we process the log parameters) then I still want to use our logging format.

File: relay/cmd/main.go (hunk 3 above)
Comment (lixmal) [reply relay/cmd/main.go:89]: Apologies, indeed you are right. I misinterpreted how we visit the flags

**Review by lixmal — APPROVED** [review]

File: encryption/route53.go [hunk 4]
```diff
@@ -23,14 +25,17 @@ type Route53TLS struct {
...
 	certmagic.Default.Logger = logger()
 	certmagic.Default.Storage = &certmagic.FileStorage{Path: r.DataDir}
 	certmagic.DefaultACME.Agreed = true
-
 	if r.Email == "" {
-		certmagic.DefaultACME.Email = "myemail@example.com"
+		certmagic.DefaultACME.Email = emailFromDomain(r.Domains[0])
 	} else {
-		certmagic.DefaultACME.Email = r.Email
+		certmagic.DefaultACME.Email = ""
```
Comment (mlsmaycon) [inline encryption/route53.go]: should be `= r.Email`

**Review by mlsmaycon — APPROVED** [review]

**Pull request merged**